packages/admin/package.json

```diff
@@ -8,6 +8,7 @@
     ],
     "dependencies": {
         "@material-ui/lab": "^4.0.0-alpha.57",
+        "@comet/admin-icons": "^2.0.0-alpha.7",
```

Comment:
Is this correct under `dependencies`?  
I tried it with `peerDependencies` and got an error with `module "@comet/admin-icons" not found`...

Reply:
I don't think it will be a problem to have it as dependency, as the app will use the same version. And even if not - in the worst case we have two different versions - and that won't cause errors ("just" increase package size)